src/gui-wizard-gtk/wizard.c

```diff
@@ -1966,7 +1966,7 @@ static bool check_minimal_bt_rating(const char  *event_name,
 
     if (!event_name)
         error_msg_and_die(_("Cannot check backtrace rating because of invalid event name"));
-    else if (libreport_prefixcmp(event_name, "report") != 0)
+    else if (g_str_has_prefix(event_name, "report") == 0)
```

Comment:
I would find `!g_str_has_prefix` slightly clearer. (+ in all the other occurrences.)

Reply:
But you told me to use explicit `== 0` with `strcmp` just the other day! :)

Reply:
Well, `g_str_has_prefix` return a Boolean (`0` or `1`) while `strcmp` may return `-1`, `1` or `0`, so the semantics are slightly different. I mean, it's not a big deal, it's just my point of view on these syntactic matters.

Reply:
Oh, good point. Never thought of that. Probably because I don't remember ever seeing the distinction between `strcmp` returning 1 and -1 put to actual use.